go.mod

```diff
@@ -0,0 +1,17 @@
+module github.com/kubewarden/go-policy-template
+
+go 1.20
+
+require (
+	github.com/francoispqt/onelog v0.0.0-20190306043706-8c2bb31b10a4
+	github.com/kubewarden/k8s-objects v1.27.0-kw2
```

Comment:
I think we can drop this dependency

Reply:
run go mod tidy, it stays as indirect as it is needed by the kw sdk